.github/workflows/pr-checks.yml

```
@@ -0,0 +1,109 @@
+        run: npm run build
+        env:
+          NODE_ENV: production
+
+      - name: Serve built SPA
+        run: npx serve .output/public -l 3000 &
+
+      - name: Wait for server
+        run: npx wait-on http://localhost:3000 --timeout 15000
+
+      - name: Run Lighthouse CI
+        uses: treosh/lighthouse-ci-action@v10
```

actionlint (1.7.10)

the runner of "treosh/lighthouse-ci-action@v10" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)